doc/classes/Viewport.xml

```diff
@@ -144,13 +144,13 @@
 		<method name="gui_get_focus_owner" qualifiers="const">
 			<return type="Control" />
 			<description>
-				Returns the [Control] having the focus within this viewport. If no [Control] has the focus, returns null.
+				Returns the [Control] having the focus within this viewport. If no [Control] has the focus, returns [code]null[/code].
```

Comment:
```suggestion
				Returns the currently focused [Control] within this viewport. If no [Control] is focused, returns [code]null[/code].
```
"Has the focus" / "having the focus" seems awkward. This page already uses "currently focused" in other members.

Reply:
Both are used across the class reference. I prefer the latter too, of course.

Reply:
My opinion as a native English speaker is that "currently focused control", "focused control", "if the control is focused",  and *maybe* "if the control has focus" sound natural and grammatical to me. However, "if the control has the focus" and "control having the focus" might not be technically wrong but are definitely not idiomatic English. I could maybe see those two forms being easier for if English is not your first language, though.

I would personally try to move towards the first set of phrases if changes are already being made to a string.